dev-python/jsonschema/jsonschema-4.3.3-r1.ebuild

```
@@ -0,0 +1,52 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DISTUTILS_USE_SETUPTOOLS=pyproject.toml
+PYTHON_COMPAT=( python3_{8..10} )
+inherit distutils-r1
+
+DESCRIPTION="An implementation of JSON-Schema validation for Python"
+HOMEPAGE="https://pypi.org/project/jsonschema/ https://github.com/Julian/jsonschema"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~mips ~ppc ~ppc64 ~riscv ~s390 ~sparc ~x86"
+
+RDEPEND="
+	dev-python/attrs[${PYTHON_USEDEP}]
+	>=dev-python/pyrsistent-0.18.0[${PYTHON_USEDEP}]
+	$(python_gen_cond_dep '
+		>=dev-python/importlib_resources-1.4.0[${PYTHON_USEDEP}]
+	' pypy3 python3_8)
```

Please use here the new syntax, as of [here](https://projects.gentoo.org/python/guide/depend.html#dependencies-conditional-to-python-version) (variant 3)

```suggestion
	' 3.8)
```